IBPSA/Fluid/HeatExchangers/BaseClasses/PartialEffectivenessNTU.mo

```diff
@@ -105,6 +105,11 @@ initial equation
     Q_flow_nominal = -m2_flow_nominal*cp2_nominal*(T_a2_nominal - T_b2_nominal);
     eps_nominal = abs(Q_flow_nominal/((T_a1_nominal - T_a2_nominal)*
       CMin_flow_nominal));
+    assert(Q_flow_nominal / (T_a1_nominal - T_a2_nominal) >= 0,
+    "In " + getInstanceName() + ": Q_flow_nominal is defined with the wrong sign. " +
+    "By convention, a positive value describes a heat flow from port 1 to port 2. " +
```

Comment:
I suggest `By convention, a positive value describes a heat flow from Medium1 to Medium2`, or from fluid 1 to fluid 2.

Reply:
Done.